site/js/modules/webui/render.js

```diff
@@ -165,7 +165,7 @@ function makePage(
     });
 
     container.querySelector(".user-input-fields fieldset")?.addEventListener("keydown", function(event) {
-        if (event.key != "Enter") {
+        if (event.key != "Enter" || event.target.tagName === "TEXTAREA") {
```

Comment:
In most languages (and *especially* in JS),
I recommend using extra parens to clear up order of operations and group expressions.
JS can surprise you in some cases
(like `&&` having higher (not the same) precedence than `||`, so `a || b && c` != `(a || b) && c`).
So here, I recommend:
```
if ((event.key != "Enter") || (event.target.tagName === "TEXTAREA")) {
```